src/server/ping.js

```diff
@@ -23,7 +23,7 @@ module.exports = function (client, server, { beforePing = null, version }) {
         online: server.playerCount,
         sample: []
       },
-      description: { text: server.motd },
+      description: server.motd,
```

Comment:
motdMsg makes more sense, otherwise sounds good

Reply:
> This is a breaking change as you can see by the failing tests. Maybe add something like 
> ```js
> description: server.rawMotd ?? { text: server.motd }
> ```
> so that the code is backwards compatible 

If we ignore the third commit, it already works